crates/storage/nippy-jar/src/lib.rs

```diff
@@ -21,24 +24,24 @@ pub struct NippyJar {
     columns: usize,
     /// Compressor if required
     compressor: Option<Compressors>,
+    /// Filter
+    filter: Option<Filters>,
     #[serde(skip)]
     /// Data path for file. Index file will be `{path}.idx`
     path: Option<PathBuf>,
     /// soon
-    bloom_filter: bool,
-    /// soon
     phf: bool,
 }
 
 impl NippyJar {
     /// Creates new [`NippyJar`].
-    pub fn new(columns: usize, bloom_filter: bool, phf: bool, path: &Path) -> Self {
+    pub fn new(columns: usize, phf: bool, path: &Path) -> Self {
```

Comment:
nit would make this `with_phf(bool)` if we expect it to be false most of the time

Reply:
overall (parent PR) it's still a draft/wip, it will be removed eventually